specification/common-types/resource-management/v3/privatelinks.json

```diff
@@ -36,6 +36,14 @@
     "PrivateEndpointConnectionProperties": {
       "type": "object",
       "properties": {
+        "groupIds": {
+          "description": "The private endpoint resource group ids.",
```

Comment:
this description should indicate what these group IDs actually do (yes, the equivalent "groupid" description below is also bad but this is new)

Reply:
this is a tough one. i looked through the networking team's swagger to see if i could find a better description, but i don't see anything more specific about these ids there.

@pilor

Reply:
I'd be happy to make a V4 version of the common spec if that's heading in the right direction for all Azure teams. Should teams be moving towards using common definitions?

Reply:
yes, they should.  Go for it if you are willing to do that